OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -595,16 +604,200 @@ private static void BuildDocument(List<Source> sources, WordprocessingDocument o
                         }
                     }
                     ++sourceNum;
+                }
+                if (sources.Any(s => s.KeepSections) && !output.MainDocumentPart.GetXDocument().Root.Descendants(W.sectPr).Any())
+                {
+                    using (OpenXmlMemoryStreamDocument streamDoc = new OpenXmlMemoryStreamDocument(sources[0].WmlDocument))
```

Comment:
Here you're accessing `sources[0]` but you test for `sources.Any(s=>s.KeepSections)`. Are you wanting to access the first of the `KeepSections`?

Reply:
Actually, what is happening here is that the code is looking to see if *any* of the sources have KeepSections set to true, and then if so, pulling the default section properties from the first document.  The code is doing what I intended - this is how I defined the module to work.

Reply:
Cool - just wanted to make sure :)